rules/gandalf.js

```diff
@@ -184,6 +184,7 @@ module.exports = {
       'exports': 'never',
       'functions': 'never'
     }],
+    "max-params": ["error", 3],
```

Comment:
Change the double quotes to single quotes.

Reply:
fixed